hack/fetch-releases.sh

```diff
@@ -379,6 +379,14 @@ main() {
   if [[ ${TARGET} == "openshift" ]]
   then
       release_yaml results release_base 00-results ${r_version}
+      # Add PostgreSQL manifest to main directory for OpenShift
+      # PostgreSQL is now part of main reconciliation flow, not PreReconcile
+      results_version_dir=${ko_data}/tekton-results/${r_version//v}
+      if [[ -f "${ko_data}/static/tekton-results/internal-db/db.yaml" ]]; then
+        echo "Copying PostgreSQL manifest to main results directory..."
```

Comment:
Instead of copying the PostgreSQL manifests into the main manifest location, we should append them to the main reconciler manifest. cc: @khrm